src/fsharp/FSharp.Compiler/FSharp.Compiler.fsproj

```diff
@@ -11,7 +11,7 @@
     <OutputType>Library</OutputType>
     <AssemblyName>FSharp.Compiler</AssemblyName>
     <DefineConstants>EXTENSIONTYPING;COMPILER;INCLUDE_METADATA_READER;INCLUDE_METADATA_WRITER;EXTENSIBLE_DUMPER;$(DefineConstants)</DefineConstants>
-    <NoWarn>$(NoWarn);62;9</NoWarn>
+    <NoWarn>$(NoWarn);62;9;3180</NoWarn>
```

Comment:
in debug build i get an error because 'mutable captured by closure' warning ( but with warnings as error => error)
why doesnt that happen on appveyor or appveyor.cmd? warnaserror is true